sqlmesh/core/model/decorator.py

```diff
@@ -12,9 +12,11 @@
 from sqlmesh.core.macros import MacroRegistry
 from sqlmesh.utils.jinja import JinjaMacroRegistry
 from sqlmesh.core import constants as c
-from sqlmesh.core.dialect import MacroFunc, parse_one
+from sqlmesh.core.dialect import SQLMESH_MACRO_PREFIX, MacroFunc, parse_one
+from sqlmesh.core import dialect as d
 from sqlmesh.core.model.definition import (
     Model,
+    _meta_renderer,
```

Comment:
Should we rename this to get rid of the leading `_`? It's not a private helper anymore if we reference it here.